decimal64math_test.go

```diff
@@ -379,3 +379,93 @@ func BenchmarkDecimal64Sub(b *testing.B) {
 		_ = x[i%len(x)].Sub(y[i%len(y)])
 	}
 }
+
+func TestAddOverflow(t *testing.T) {
+	require := require.New(t)
+
+	a := MustParseDecimal64("-9.999999999999999e384") // largest dec64 number
+	b := MustParseDecimal64("0.000000000000001e384")
+	require.Equal(NegInfinity64, a.Sub(b))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0.000000000000001e384")
+	require.Equal(Infinity64, a.Add(b))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0.0000000000000001e384")
+	require.Equal(a, b.Add(a))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0")
```

Comment:
Use `Zero64`. And don't bother sticking it in a variable. Just use it directly in `require.Equal(…)`. Ditto below.